test/unit/game.test.js

```diff
@@ -0,0 +1,20 @@
+const Game = require('../../lib/game');
+
+describe('Game class', () => {
+  
+  test('calculateScore method should return the current score', () => {
+    const bowling = new Game;
+    bowling.roll(4);
+    bowling.roll(5);
+    expect(bowling.calculateScore()).toBe(9);
+  });
+  
+  test('checkFrame should increase current frame if the frame current frame has 2 rolls', () => {
+    const bowling = new Game;
+    bowling.roll(4);
+    bowling.roll(5);
+    expect(bowling.currentFrame).toBe(0);
```

Comment:
I may be completely wrong, but toBe sometimes fails for me when I expect something like the current frame toBe 0
So I started using toEqual instead, I haven't read much into it but I think it's more appropriate for when your expecting a value.

toBe may be more appropriate for when you expect something toBe nil or toBe true